FCCee/ALLEGRO/compact/ALLEGRO_o1_v03/ALLEGRO_o1_v03.xml

```diff
@@ -33,14 +33,79 @@
   <include ref="Beampipe.xml"/>
   <include ref="BeamInstrumentation.xml"/>
   <include ref="LumiCal.xml"/>
-  <include ref="HOMAbsorber.xml"/> 
-  <!--taken from DetFCCeeCLD/compact/FCCee_o2_v02  -->
-  <include ref="Vertex.xml"/>
-  <include ref="DriftChamber.xml"/>
-  <include ref="ECalBarrel_thetamodulemerged.xml"/>
+  <include ref="HOMAbsorber.xml"/>
+  <include ref="Vertex_IDEA_o1_v01.xml"/> <!-- symbolic link to ../../../IDEA/compact/IDEA_o1_v03/Vertex_IDEA_o1_v01.xml"/>-->
+  <include ref="DriftChamber_o1_v02.xml"/> <!-- symbolic link to ../../../IDEA/compact/IDEA_o1_v03/DriftChamber_o1_v02.xml -->
+  <include ref="ECalBarrel_thetamodulemerged.xml"/> <!-- if you remove the ECalBarrel, you also have to remove or update the "GlobalSolenoid" field (it depends on ECAL dimensions) -->
   <include ref="HCalBarrel_TileCal.xml"/>
   <include ref="ECalEndcaps_coneCryo.xml"/>
   <include ref="HCalEndcaps_ThreeParts_TileCal.xml"/>
   <include ref="MuonTagger.xml"/>
 
+  <fields>
+    <!-- FIXME this is a place holder to get a reasonable magnetic field, it should be a real MAP obtained from FEM tools taking into account HCAL return yoke -->
+    <!-- The z extent should also accomodate space for the feedthroughs, need input from R&D and decisions on cold/warm electronics -->
+    <field name="GlobalSolenoid" type="solenoid"
+           inner_field="SolenoidField"
+           outer_field="-1.0*tesla"
+           zmax="EMBarrel_dz"
+           outer_radius="BarCryoECal_rmax-CryoBarrelBackWarm-CryoBarrelBackCold">
```

Comment:
should this also include a -NLiqBathThicknessBack ?
In any case, this is so small, and in both cases the outer_radius will be greater than that of the LAr volume, so it should have no practical consequences.

Reply:
I would'nt say so, because `NLiqBathThicknessBack` is for services right after the sensitive calo, the solenoid is after that. Of course we are missing a detailed mechanical design here (and the software implementation of the cryostat + solenoid is just accounting for material budget, not for precise positions of the components) but indeed the impact for what we do at this point should be invisible. 

![Screenshot 2024-06-06 at 4 00 05 PM](https://github.com/key4hep/k4geo/assets/3646269/a6aeb7d0-c157-4973-b3b8-c63cff70c21d)

Reply:
ah, ok, thanks! I wasn't sure whether the NLiq was where you pointed at, or was the liquid nitrogen inside the cryostat, between the warm and cold parts.